### PR TITLE
create nanoly endpoint

### PR DESCRIPTION
**Pull request by Crypto-One-dev**
EndPoint
{{API URL}}/nanoly?key=harvest-key

### CODE REVIEW

**Comment by Crypto-One-dev** [comment]
move some vaults to inactive

File: data/mainnet/tokens.js [hunk 1]
```diff
@@ -8035,7 +8041,8 @@ module.exports = {
     testInactive: true,
     chain: CHAINS_ID.ETH_MAINNET,
     category: VAULT_CATEGORIES_IDS.INACTIVE,
-    displayName: 'Uniswap V3: ETH-DPI',
+    displayName: 'ETH-DPI',
```
[the remark this answers is not in the document]
Comment (Crypto-One-dev) [reply data/mainnet/tokens.js:8044]: @CryptJS13 
I don't understand for this comment. What should I do exactly?

[the remark this answers is not in the document]
Comment (CryptJS13) [reply data/mainnet/tokens.js:8044]: It is not an active vault, so it should not show up on your new endpoint. So you should either add the `inactive` tag here, or check for the `testInactive` flag

[the remark this answers is not in the document]
Comment (Crypto-One-dev) [reply data/mainnet/tokens.js:8044]: Okay, got it

**Comment by CryptJS13** [comment]
@Crypto-One-dev Code looks good to me. If the test runs properly and output looks like it should I think this is good to merge.

**Comment by Crypto-One-dev** [comment]
@CryptJS13 I tested it and output looks fine.

**Pull request merged**